testng-eclipse-plugin/src/main/org/testng/eclipse/util/LaunchUtil.java

```diff
@@ -130,7 +130,7 @@ private static void launchSuiteConfiguration(IProject project, String fileConfNa
       String suiteFilePath,
 		  String mode, ILaunchConfiguration prevConfig, Set failureDescriptions) {
     ILaunchConfigurationWorkingCopy configWC =
-        createLaunchConfiguration(project, fileConfName, null);
+        createLaunchConfiguration(project, project.getName() + "_" + fileConfName, null);

```

Comment:
testng issue #820: prefix the launch configuration name with project name so that the launch conf name is unique at workspace level